integrationtests/volume_test.go

```diff
@@ -119,35 +122,12 @@ func negativeVolumeTests(t *testing.T) {
 	runNegativeVolumeStatsRequest(t, client, "-1")
 }
 
-// sizeIsAround returns true if the actual size is around the expected size
-// (considers the fact that some bytes were lost)
-func sizeIsAround(t *testing.T, actualSize, expectedSize int64) bool {
-	// An upper bound on the number of bytes that are lost when creating or resizing a partition
-	var volumeSizeBytesLoss int64 = (20 * 1024 * 1024)
-	var lowerBound = expectedSize - volumeSizeBytesLoss
-	var upperBound = expectedSize
-	t.Logf("Checking that the size is inside the bounds: %d < (actual) %d < %d", lowerBound, actualSize, upperBound)
-	return lowerBound <= actualSize && actualSize <= upperBound
-}
-
 func negativeDiskTests(t *testing.T) {
-	var client *v1beta3client.Client
-	var err error
-
-	if client, err = v1beta3client.NewClient(); err != nil {
-		t.Fatalf("Client new error: %v", err)
-	}
-	defer client.Close()
+	_, err := volume.New(volumeapi.New())
```

Comment:
below I saw:

```
	volumeClient, err := volume.New(volumeapi.New())
```

which is the same as this test, because we're just moving things around I think it's ok to keep it in this PR but we should remove this test in a later PR when things get moved and stable